CorsixTH/Src/th_movie.cpp

```diff
@@ -136,9 +116,8 @@ THMoviePictureBuffer::~THMoviePictureBuffer()
 void THMoviePictureBuffer::abort()
 {
     m_fAborting = true;
-    SDL_LockMutex(m_pMutex);
-    SDL_CondSignal(m_pCond);
-    SDL_UnlockMutex(m_pMutex);
+    std::lock_guard<std::mutex> lock(m_mutex);
+    m_cond.notify_all();
```

Comment:
SDL woke up 1 thread, but  I guess this is fine, as everybody will find the m_fAborting flag, and quit?